### DISCUSSION TITLE
How to put the SHAPE of the variable at the front of the variable information during debugging？

### DISCUSSION THREAD

**Discussion by chenhui2016** [microsoft/vscode-discussions#1300] [Extension Development QnA] [answered]
This is the information of a certain variable during debugging. If I want to see the shape of the variable, I need to search backwards, which is very troublesome. 
How to put the SHAPE of the variable at the front of the variable information during debugging？
![6ddd2568540f2ca8fa147821d3737435](https://github.com/user-attachments/assets/86d579df-caff-45f1-9c17-a1eeb141ac1c)

**Comment by isidorn** [comment, marked as answer]
This is a community for VS Code extension developers. For general vscode questions please ask on stack overflow
